packages/devops/scripts/ci/setupEnvironmentVariables.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -x
+DIR=$(dirname "$0")
+
+# packages needing .env during CI/CD are all deployable, plus auth and data-api
```

Comment:
Minor: move to a getPackagesWithEnvVars.sh?

Reply:
Maaybe...there are other packages that can possibly take env vars I believe, but are only needed for some local things like cypress testing. I'll think on this.